template_content/__test__/{{ project_name }}.test.ts.jinja

```diff
@@ -18,6 +18,10 @@ describe('{{ camel_case_name }}', () => {
     await fixture.beforeEach();
     const { algod, testAccount } = fixture.context;
     const sender = algosdk.generateAccount();
+    await algokit.ensureFunded({
```

Comment:
I forget why we define this sender account in the first place... `testAccount` is used in the app client which is guaranteed to be funded via the fixture.